test/README.md

```diff
@@ -1,2 +1,15 @@
-This directory should contain scripts and files needed to test your module's code.
- 
+The test data exists in two archives in order to speed up testing:
+../data/OrthoFinder_Test_Reference.tar.gz
+../data/OrthoFinder_Test_Result_Reference.tar.gz
+
+The first archive is for running with OrthoFinder and the second
+archive is for running without OrthoFinder (i.e. it already contains
+results from OrthoFinder). You set which file it is in the deploy.cfg,
+in the test_data field, and you copy it to the ../test_local/refdata
+directory.
+
+It has to be unpacked. It should be unpacked, for testing purposes, in
+../scripts/entrypoint.sh but I haven't figure out how to read
+deploy.cfg in there yet. Also, the testing within the kb_orthofinder
```

Comment:
I think you already figured it out by grepping it right?

Reply:
I thought that was an ugly hack, not sure of a proper way to do this in bash

Reply:
I'm not sure if anyone does it in kbase. But the first thing that comes to mind is a file with environmental variables and loading it in, so those variables are available in memory, so you won't have to do any greps.